### DISCUSSION TITLE
Thank You Matte

### DISCUSSION THREAD

**Discussion by nuwanz** [MattieBelt/mattie-strapi-bundle#102] [General]
Thank you matte for this amazing work.

**Comment by jordanvector** [comment]
I would like to also say thank you for this amazing work!  The Strapiv4 + Algolia integration is a big part of a new project we are working on internally and was wondering if Matte would be interested in a sponsorship to finish some upcoming features here: https://mattie-bundle.mattiebelt.com/search/plugin#future-features

Just curious and thank you for all of the work done here.

**Reply by MattieBelt** [reply]
Hey @jordanvector, thanks for the appreciation, I wondered if you were still up for a sponsorship as I have some free time coming up. Which feature would you like to see added to the plugin? Feel free to reach out via mattiebelt@gmail.com or via the Discord community server.

**Reply by jordanvector** [reply]
Hey @MattieBelt definitely still interested. A few high level things we were interested in
1.  Manage indexes from admin panel (e.g. View or Delete indexed objects).  There does not seem to be a way (or I have not figured it out yet) to mass update search indexes from the strapi admin, both mass adding of items to index or removing.
2. Some sort of admin interface to configure or view the algolia indexes
3. Also liked this idea a lot - Global search bar in admin panel (e.g. InstantSearch.js)

I can email directly if that works better for you.

Cheers.

**Reply by MattieBelt** [reply]
Hi @jordanvector,

Thanks for coming back to me and good to hear you are still interested.
Yes please, let's discuss this via email, you can best contact me at mattiebelt@gmail.com.
Feel free to propose how you envisioned this and how I best can help you to develop these new features for the plugin.
